.github/workflows/ci.yaml

```diff
@@ -89,6 +89,14 @@ jobs:
           sudo apt-get install graphviz
           pip install tox
           tox -e docs
+      # NOTE(sileht): workflow run on/pull_request doesn't have access to
+      # secrets
+      # So, we upload it as artefact, then another workflow docs-pr.yaml will
+      # upload it on gcp.
+      - uses: actions/upload-artifact@v2
+        with:
+          name: docs-pr-${{ github.event.pull_request.number }}
+          path: docs/build
```

Comment:
?

Reply:
I don't get why this show up here, I just rebase/push and it disapears.